drivers/impedance-analyzer/ad5933/ad5933.h

```diff
@@ -143,6 +143,11 @@ struct ad5933_init_param {
 	uint16_t current_settling;
 };
 
+typedef struct ad5933_impedance {
```

Comment:
Try to avoid typedefs. https://www.kernel.org/doc/html/v4.10/process/coding-style.html#typedefs